Hardware/Star64/index.md

```diff
@@ -0,0 +1,211 @@
+---
+riscv_hardware: true
+cmake_plat: star64
+xcompiler_arg: -DRISCV64=1 -DUseRiscVOpenSBI=OFF
+platform: Pine64 Star64
+arch: RV64GBC, RV64IMAC, RV32IMAFBC
+virtualization: "No"
+iommu: "No"
+simulation_target: false
+Status: "Unverified"
+Contrib: "Community"
+Maintained: "seL4 Foundation"
+soc: StarFive JH7110
+cpu:  U74-MC, E24
+SPDX-License-Identifier: CC-BY-SA-4.0
+SPDX-FileCopyrightText: 2020 seL4 Project a Series of LF Projects, LLC.
+---
+
+# Pine64 Star64
+
+The Star64 is a RISC-V SBC by Pine64 based on the StarFive JH7110 SoC. Check
+[here](https://wiki.pine64.org/wiki/STAR64) for details.
+
+The StarFive JH7110 SoC is comprised of the SiFive U74-MC (four 64-bit U74
+cores and one 64-bit S7 core) and one SiFive E24 32-bit core. Note that by
+default seL4 is setup to use the U74 cores as neither the S7 or E24 core has
+S-Mode, which seL4 requires.
+
+The Star64 arrives with the following boot process from the SPI flash:
+
+1. U-Boot SPL starts
+2. OpenSBI starts
+3. U-Boot proper starts
+
+From U-Boot proper you can then load and start an seL4 image, see below for details.
+
+## Building the GCC toolchain
+
+{% include risc-v.md %}
+
+## Building seL4test
+
+{% include sel4test.md %}
```

Comment:
The only thing left to do is specify that if you have the 8GB model then you can use the `overlay-8gb.dts`.